src/debugger/mock-ui/debugger-cli.js

```diff
@@ -33,11 +35,23 @@ function run(process, console) {
       adapterPath = args.shift();
     } else if (arg === "prepack") {
       prepackCommand = args.shift();
+    } else if (arg === "inFilePath") {
```

Comment:
Take out these block of arguments parsing code into its own function as self-commenting.